scripts/deribit_atm_greeks_collector.py

```
@@ -0,0 +1,317 @@
+    global RUNNING
+    RUNNING = False
+    print(f"[deribit-greeks] received signal={signum}, stopping...", flush=True)
+
+
+def _run_psql(sql: str, at_mode: bool = False) -> str:
+    cmd = [PSQL_BIN, DB_URL, "-v", "ON_ERROR_STOP=1", "-X"]
+    if at_mode:
+        cmd.extend(["-A", "-t", "-F", "\t"])
+    cmd.extend(["-c", sql])
+
+    proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)